sdk/search/azure-search-documents/azure/search/documents/_utils.py

```diff
@@ -3,6 +3,14 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+from azure.core.pipeline import policies
+
+CREDENTIAL_SCOPES = ['https://search.azure.com/.default']
+
 def is_retryable_status_code(status_code):
     # type: (int) -> bool
     return status_code in [422, 409, 503]
+
+def get_authentication_policy(credential):
+    authentication_policy = policies.BearerTokenCredentialPolicy(credential, CREDENTIAL_SCOPES)
```

Comment:
is it possible that a user might want to pass their own scopes? For different clouds, e.g. govt or mooncake? Wondering if scopes should be a kwarg

Reply:
For now (at least in this preview), no. It is hard-coded by the search team.